src/main/java/ilex/util/ThreadLogger.java

```diff
@@ -136,7 +136,7 @@ public void close( )
 	 * its own priority filtering.
 	 */
 	@Override
-	public synchronized void log( int priority, String text )
```

Comment:
does slf4j take care of thread sharing, if synchronized is removed here?

Reply:
For where it's used yes. otherwise that synchronization should happen exactly at point of writing.
But I'm fairly certain it isn't necessary. It's going to require a bit more testing for sure, but having synchronized here was the cause of a dead lock. due to the user of ilex.util.Logger within the DacqEventDao; it's possible migrating that class to slf4j would allow synchronize to go back, but to be blunt having synchronized on a log statement is asking for performance problems.

This shouldn't get merged in before the next release though so we have some time.